src/technologies/u.json

```diff
@@ -1022,6 +1022,19 @@
     },
     "website": "https://user.com"
   },
+  "Utiq": {
+    "cats": [
+      36
+    ],
+    "description": "Utiq empowers brands and publishers to address audiences through Telco-powered first-party identifier.",
+    "icon": "Utiq.svg",
+    "oss": false,
+    "website": "https://utiq.com/",
+    "xhr": "/utiqLoader\\.js",
```

Comment:
I think this only works for hostnames: https://github.com/HTTPArchive/wappalyzer#:~:text=Hostnames%20of%20XHR%20requests.

Reply:
Thanks.
Of course, JS script is detectable with `scriptSrc` and not `xhr`.